allensdk/api/cloud_cache/manifest.py

```diff
@@ -71,8 +71,8 @@ def load(self, json_input):
             raise ValueError("Expected to deserialize manifest into a dict; "
                              f"instead got {type(self._data)}")
 
-        self._version = copy.deepcopy(self._data['dataset_version'])
-        self._file_id_column = copy.deepcopy(self._data['file_id_column'])
+        self._version = copy.deepcopy(self._data['manifest_version'])
```

Comment:
Maybe just do 1 deepcopy instead of two?

Reply:
Are you concerned that these deep copies are each copying the entire `self._data` dict? I thought I was only copying the relevant elements of `self._data` (I admit that is a little paranoid).

Reply:
(I'm going to merge for the sake of having this available for CLI development, but I do want to understand your concern)